data/mainnet/addresses.json

```diff
@@ -1778,6 +1786,30 @@
         "NewStrategy": "0xA87Abed957338A5dE2106D13d1967cF62d6d7996",
         "NewPool": "0xb42E1201cc041727B063cfbB506Ca7ee74B7857e"
       },
+      "jarvis_SES_2JPY": {
```

Comment:
I think all these addresses are already in this file, so no need to add them. Make sure you pull the latest version of `master` into your working branch.